tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py36,docs
+envlist = py3,docs
 
 [travis]
 python =
```

Comment:
I am in favor of adding Python 3.7 and 3.8 here to reduce the possibility of differences between a local run of tox and a Travis build.

Reply:
I added testing for 3.7 and 3.8 to Travis. There could still be differences between local runs and travis builds: If 3.6 passes, which I use locally, but 3.7 fails, then I only realize that after building with Travis. 

But before the change we could have encountered problems the other way round: Local tests with 3.7 fail, while 3.6 passes on Travis. This way at least we would realize when we don't support a Python version.

I guess there will always be a compromise on some level, between "thorough local testing that mimics Travis" and "easy simple local testing via `tox`".